component/local/file/file.go

```diff
@@ -98,11 +101,19 @@ func New(o component.Options, args Arguments) (*Component, error) {
 		opts: o,
 
 		reloadCh: make(chan struct{}, 1),
+		lastAccessed: prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "local_file_timestamp_last_accessed_unix_seconds",
```

Comment:
Don't we want this with `agent` or `agent_component` namespace?

Reply:
`agent_` sounds reasonable